site/kube-addon.md

```diff
@@ -65,6 +65,17 @@ Kubernetes see the
 and the [NetworkPolicy API object
 definition](http://kubernetes.io/docs/api-reference/extensions/v1beta1/definitions/#_v1beta1_networkpolicy).
 
+### Dropped connections
+
+Details on connections dropped by the Network Policy Controller can be found in the `weave-npc` container's logs:
```

Comment:
I had this initially, but wasn't sure if appropriate / useful, so removed it before pushing.
I will add it back. Thanks!